File: programme/reverse-of-a-number/ReverseOfNumber.java
// Write a program to print the reverse of a number
import java.util.Scanner;

public class ReverseOfNumber {
    public static void main(String[] args) {
        int a, n;
        Scanner sc = new Scanner(System.in);
        System.out.print("Input  : ");
        a = sc.nextInt();
        System.out.print("Output : ");
        while (a > 0) {
            n = a % 10;
            System.out.print(n);
            a /= 10;
        }
        System.out.println();
        sc.close();
    }
}

File: programme/reverse-of-a-number/reverse-of-number.js
// Write a programmeto print reverse of a number

"use strict";

// perform following command to run programme
// node reverse-of-a-number.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", (num) => {
  num = parseInt(num.split("").reverse().join(""));
  if (isNaN(num)) {
    console.error("Some error occured");
  } else {
    console.log(`Output : ${num}`);
  }
  rl.close();
});
